Compilation failed when _BLE_TRACE_ enabled

```
.pio/libdeps/nano_33_iot/ArduinoBLE/src/utility/L2CAPSignaling.cpp: In member function 'virtual void L2CAPSignalingClass::handleSecurityData(uint16_t, uint8_t, uint8_t*)':
.pio/libdeps/nano_33_iot/ArduinoBLE/src/utility/L2CAPSignaling.cpp:233:21: error: 'X' was not declared in this scope
     btct.printBytes(X,16);
                     ^
.pio/libdeps/nano_33_iot/ArduinoBLE/src/utility/L2CAPSignaling.cpp:235:21: error: 'Y' was not declared in this scope
     btct.printBytes(Y,16);
                     ^
.pio/libdeps/nano_33_iot/ArduinoBLE/src/utility/L2CAPSignaling.cpp: In member function 'virtual void L2CAPSignalingClass::smCalculateLTKandConfirm(uint16_t, uint8_t*)':
.pio/libdeps/nano_33_iot/ArduinoBLE/src/utility/L2CAPSignaling.cpp:416:19: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     for(int i=0; i<sizeof(Eb); i++){
                  ~^~~~~~~~
*** [.pio/build/nano_33_iot/lib42d/ArduinoBLE/utility/L2CAPSignaling.cpp.o] Error 1
```

--- a/src/utility/L2CAPSignaling.cpp
+++ b/src/utility/L2CAPSignaling.cpp
@@ -230,9 +230,9 @@ void L2CAPSignalingClass::handleSecurityData(uint16_t connectionHandle, uint8_t 
     Serial.print("V      : ");
     btct.printBytes(V,32);
     Serial.print("X      : ");
-    btct.printBytes(X,16);
+    btct.printBytes(HCI.Na,16);
     Serial.print("Y      : ");
-    btct.printBytes(Y,16);
+    btct.printBytes(HCI.Nb,16);
     Serial.print("g2res  : ");
     btct.printBytes(g2Result,4);
     Serial.print("Result : ");
@@ -413,7 +413,7 @@ void L2CAPSignalingClass::smCalculateLTKandConfirm(uint16_t handle, uint8_t expe
     // Send our confirmation value to complete authentication stage 2
     uint8_t ret[17];
     ret[0] = CONNECTION_PAIRING_DHKEY_CHECK;
-    for(int i=0; i<sizeof(Eb); i++){
+    for(uint i=0; i<sizeof(Eb); i++){
       ret[sizeof(Eb)-i] = Eb[i];
     }
     HCI.sendAclPkt(handle, SECURITY_CID, sizeof(ret), ret );

--- a/src/utility/L2CAPSignaling.cpp
+++ b/src/utility/L2CAPSignaling.cpp
@@ -413,7 +413,7 @@ void L2CAPSignalingClass::smCalculateLTKandConfirm(uint16_t handle, uint8_t expe
     // Send our confirmation value to complete authentication stage 2
     uint8_t ret[17];
     ret[0] = CONNECTION_PAIRING_DHKEY_CHECK;
-    for(uint i=0; i<sizeof(Eb); i++){
+    for(uint32_t i=0; i<sizeof(Eb); i++){
       ret[sizeof(Eb)-i] = Eb[i];
     }
     HCI.sendAclPkt(handle, SECURITY_CID, sizeof(ret), ret );

